tools/ci_static_check_reports/unit_tests/test_detect_changed_modules.py

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+from typing import List, Set
+
+import pytest
+
+from ..detect_changed_modules import list_changed_modules
+
+
+@pytest.mark.parametrize(
+    "changed_files,changed_modules",
```

Comment:
Please add negative tests too:
1) input array is empty
2) non-python connectors
3) some JavaScript files
4) destination connector

Reply:
sure, will do it